cmake/public/LoadHIP.cmake

```diff
@@ -151,16 +155,55 @@ if(HIP_FOUND)
   set(PYTORCH_FOUND_HIP TRUE)
 
   # Find ROCM version for checks
-  file(READ "${ROCM_PATH}/.info/version-dev" ROCM_VERSION_DEV_RAW)
-  string(REGEX MATCH "^([0-9]+)\.([0-9]+)\.([0-9]+)-.*$" ROCM_VERSION_DEV_MATCH ${ROCM_VERSION_DEV_RAW})
+  # ROCM 4.5 and later will have header api for version management
```

Comment:
@dllehr-amd Should we update this comment and the description of the PR to say "ROCm5.0", since that's when we really intend for this switch in version detection to take place?